doc/modules/permutation_importance.rst

```diff
@@ -6,15 +6,27 @@ Permutation feature importance
 
 .. currentmodule:: sklearn.inspection
 
-Permutation feature importance is a model inspection technique that can be used
-for any :term:`fitted` :term:`estimator` when the data is tabular. This is
-especially useful for non-linear or opaque :term:`estimators`. The permutation
-feature importance is defined to be the decrease in a model score when a single
-feature value is randomly shuffled [1]_. This procedure breaks the relationship
-between the feature and the target, thus the drop in the model score is
-indicative of how much the model depends on the feature. This technique
-benefits from being model agnostic and can be calculated many times with
-different permutations of the feature.
+Permutation feature importance is a model inspection technique that measures the
+contribution of each feature to a :term:`fitted` model's performance on a given
+tabular dataset. This technique is particularly useful for non-linear or opaque
+:term:`estimators`, and involves randomly shuffling the values of a single
+feature and observing the resulting degradation of the model's score [1]_. By
+breaking the relationship between the feature and the target, we can determine
+how much the model relies on such feature. One key advantage of permutation
+feature importance is that it is model-agnostic, i.e. it can be applied to any
+fitted estimator. Moreover, it can be calculated multiple times with different
+permutations of the feature, providing a robust measure of feature importance
```

Comment:
I would not use "robust" but instead phrase it that we have an estimate of the results variance.